example/bank/README.md

```diff
@@ -0,0 +1,23 @@
+## Bank Example
+
+Welcome to the GoEngine Bank. 
+
+This is a basic example of how GoEngine can be used. This application consists of the following commands:
+- [API](cmd/api) The banks HTTP API, here bank account can be opened, money can be deposited and withdrawn.
+- [report-bank](cmd/report-bank) The banks general reported, this will report the total balance across all accounts and the total amount of bank accounts.
+- [report-account](cmd/report-account) The account reporter keeps track of the average accounts debit and credit information.  
+
+### Running the example
+
+In order to run the example you need a postgres DB. The provided [docker-compose.yml](docker-compose.yml) provides this. 
```

Comment:
Maybe let's not repeat `provides` so much and rephrase this as `The provided bla bla bla contains this.` Wdyt?:)